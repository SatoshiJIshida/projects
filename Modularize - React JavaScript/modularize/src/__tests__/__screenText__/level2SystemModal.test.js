import React from "react";
import { shallow } from "enzyme";
import Level2SystemModal from "../../components/screenText/level2SystemModal";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Level2 System modal", () => {
  it("should be defined", () => {
    expect(Level2SystemModal).toBeDefined();
  });

  it("should render correctly", () => {
    const modal = shallow(<Level2SystemModal name="modal-test" />);
    expect(modal).toMatchSnapshot();
  });
});
