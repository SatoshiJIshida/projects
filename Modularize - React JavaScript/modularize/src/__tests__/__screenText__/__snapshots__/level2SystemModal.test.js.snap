// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level2 System modal should render correctly 1`] = `ShallowWrapper {}`;
